src/app/contextBroker/contextBroker.cpp

```diff
@@ -325,6 +328,13 @@ PaArgument paArgs[] =
 
 
 //
+// /v2 conv-ops 
+//
+#define ENT                EntitiesRequest
+#define ENT_COMPS_V2       2, { "v2", "entities" }
```

Comment:
What do we win with a change like that?
[ Except that the code gets more difficult to read ... ]